vm/chef/cookbooks/django/files/django-config-setup

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /opt/c2d/c2d-utils || exit 1
+
+readonly default_site="$(get_attribute_value "default_site_name")"
+readonly external_ip="$(get_external_ip)"
```

Comment:
I'd set this var as `""` so if has_external_ip is true, we don't have to call the `get_external_ip` twice